### PR TITLE
More employees in the seed

### PR DESCRIPTION
**Pull request by mjoyce91**
Adds about 1,000 more employees so that we can better test the CDO Portfolio and Available Bidders

About 50 more CDOs

### CODE REVIEW

File: src/services/employees.js [hunk 1]
```diff
@@ -198,9 +198,15 @@ const get_clients = async query => {
 
 // Maps request params to employee fields for filtering
 const get_agents_filters = (params = {}) => {
-  const { rl_cd, perdet_seq_num, hru_id } = params
+  const { 'request_params.rl_cd': rl_cd, 'request_params.perdet_seq_num': perdet_seq_num, 'request_params.hru_id': hru_id } = params
```
Comment (elizabeth-jimenez) [inline src/services/employees.js:201]: I think we can remove the perdet_seq_num stuff, i dont think we ever hit `get_agents_filters` if that's in the query

File: src/services/employees.js [hunk 2]
```diff
@@ -198,9 +198,15 @@ const get_clients = async query => {
...
   const q = {}
   if (rl_cd) q['employees_roles.code'] = rl_cd
+
+  // This is only going to return 'CDO' (not 'CDO3'). Could use the addFilter function to query by multiple values
+  if (isArray(rl_cd)) {
+    q['employees_roles.code'] = 'CDO'
+  }
+
   if (perdet_seq_num) q['employees.perdet_seq_num'] = perdet_seq_num
```
Comment (elizabeth-jimenez) [inline src/services/employees.js:210]: 🔍

**Comment by elizabeth-jimenez** [comment]
We maybe will want to play with the `pageSize` in `get_paged_employees_by_query`; Scott and I increased it crazily to get around our mock pagination issue 😬  i'm just running the mock alone, so for all I know it looks ~fine~ gucci

File: src/services/employees.js [hunk 3]
```diff
@@ -198,9 +198,15 @@ const get_clients = async query => {
...
 const get_agents_filters = (params = {}) => {
-  const { rl_cd, perdet_seq_num, hru_id } = params
+  const { 'request_params.rl_cd': rl_cd, 'request_params.perdet_seq_num': perdet_seq_num, 'request_params.hru_id': hru_id } = params
   const q = {}
   if (rl_cd) q['employees_roles.code'] = rl_cd
+
+  // This is only going to return 'CDO' (not 'CDO3'). Could use the addFilter function to query by multiple values
+  if (isArray(rl_cd)) {
+    q['employees_roles.code'] = 'CDO'
+  }
```
Comment (elizabeth-jimenez) [inline src/services/employees.js:208]: I think you said we always pass `rl_cd` in as an array, so this should be gtg

**Comment by elizabeth-jimenez** [comment]
Looks good to me; just a few comments! Here is my fake approve (want scott to also approve, in case i didnt check some relationships): 🍰 🍬 🥮 👍

File: seeds/data/employees_roles.json [hunk 4]
```diff
@@ -74,5 +74,4237 @@
...
+      "perdet_seq_num": 1427717,
+      "code": "EMP"
+  },
+  {
+      "perdet_seq_num": 8643019,
+      "code": "EMP"
+  },
+    {
+        "perdet_seq_num": 3638797,
+        "code": "CDO"
```
Comment (elizabeth-jimenez) [inline seeds/data/employees_roles.json:4084]: 😱

**Comment by scott062** [comment]
@mjoyce91 Is it worth adding employee_languages seeds for this too? Those would show up on AB and client profiles

**Comment by scott062** [comment]
This worked for me but just a couple questions and LOE?

**Comment by mjoyce91** [comment]
> @mjoyce91 Is it worth adding employee_languages seeds for this too? Those would show up on AB and client profiles

yeah that's a good idea

**Comment by mjoyce91** [comment]
@scott062 - added employee-languages

**Comment by mjoyce91** [comment]
However, Client Portfolio "language" is hard-coded, so you'll only see it on AB list

**Review by scott062 — APPROVED** [review]

File: src/services/employees.js (hunk 1 above)
Comment (mjoyce91) [reply src/services/employees.js:201]: No harm in having it, right? I believe these are exposed by WS too

Comment (elizabeth-jimenez) [reply src/services/employees.js:201]: Nope 😁

**Review by elizabeth-jimenez — APPROVED** [review]
👭 👬  dual approve!

**Pull request merged**